pkg/features/embedded/grub-config/etc/elemental/grub.cfg

```diff
@@ -90,10 +90,10 @@ menuentry "${display_name}" --id active {
 }
 
 for passive_snap in ${passive_snaps}; do
-  menuentry "${display_name} (snapshot ${passive_snap})" --id ${passive_snap} {
+  menuentry "${display_name} (snapshot ${passive_snap})" --id passive${passive_snap} ${passive_snap} {
```

Comment:
Ensuring the ID is not set to a number, as then it turns to be confusing with the fallback values, which is just a counter of the menu items, starting from `0` at top. So if a menu is chosen by ID it will not definitely be a number. If a number is used, then menu counter applies.